src/index.html

```diff
@@ -13,10 +13,99 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+        <nav>
+          <ul class="nav-item">
+            <li>
+              <a
+                class="nav-links is-active"
+                href="#"
```

Comment:
The `href` attribute is set to `#`, which is a placeholder. Consider updating it to point to the actual URL or section ID for the 'APPLE' link.